assets/js/common/forms.js

```diff
@@ -239,6 +239,48 @@ function showLocalSaveWarning() {
     }
 }
 
+function animateSaveButtons() {
+    $('.js-save-btn-form').on('submit', function(event) {
```

Comment:
Having this on the submit event causes this to override the next/previous links so that they a) start the animation and b) trigger the default form action rather than submitting. Submit even is probably the right one to anchor too but need to add a check to guard against this (probably based on the `nextBtn` / `previousBtn` name)

Reply:
Took a bit of a tweak but sorted this now.